packages/account/src/Components/poa/continue-trading-button/__tests__/continue-trading-button.spec.tsx

```diff
@@ -7,7 +7,7 @@ import { ContinueTradingButton } from '../continue-trading-button';
 
 describe('<ContinueTradingButton/>', () => {
     const history = createBrowserHistory();
-    const renderWithRouter = component => {
+    const renderWithRouter = (component: React.ReactNode) => {
```

Comment:
@yauheni-kryzhyk-deriv, will this contain content other than `React or HTML Elements`?

Reply:
here in test? I guess no.. You mean to use here ReactElement?

Reply:
Yes @yauheni-kryzhyk-deriv